src/Sentry/Platforms/Cocoa/CocoaProfilerFactory.cs

```diff
@@ -0,0 +1,26 @@
+using Sentry.Internal;
+using Sentry.Cocoa.Extensions;
+
+namespace Sentry.Cocoa;
+
+internal class CocoaProfilerFactory : ITransactionProfilerFactory
+{
+    private readonly SentryOptions _options;
+
+    internal CocoaProfilerFactory(SentryOptions options)
+    {
+        _options = options;
+    }
+
+    /// <inheritdoc />
+    public ITransactionProfiler? Start(ITransactionTracer tracer, CancellationToken cancellationToken)
+    {
+        var traceId = tracer.TraceId.ToCocoaSentryId();
+        var startTime = SentryCocoaHybridSdk.StartProfilerForTrace(traceId);
+        if (startTime == 0)
```

Comment:
Is this a normal behavior of the profiler? 
I wonder if this will be hard to diagnose when "profiling isn't working" and it's hitting this return.
Do we have logs coming in from somewhere else already?

Reply:
It should never be zero, I'll just remove the check, it's pointless